app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    line = command.split()
```

Comment:
Also you can unpack command to variables right here and than use them in code. It makes your code more readable 
```
    variable, variable, variable = command.split()
```